File: hotel/src/main/java/com/github/keraton/triptr/hotel/client/HotelProviderClientV2.java
package com.github.keraton.triptr.hotel.client;

import com.github.keraton.triptr.hotel.model.Hotel;

import java.util.List;

import static java.util.Arrays.asList;

public class HotelProviderClientV2 {

    private static final String HILTON = "Hilton";
    private static final String NOVOTEL = "Novotel";
    private static final String IBIS = "Ibis";

    public List<Hotel> find(String city) {
        return asList(
                new Hotel("H001", HILTON, city, 5),
                new Hotel("H002", NOVOTEL, city, 3),
                new Hotel("H003", IBIS, city, 2)
        );
    }
}

File: hotel/src/main/java/com/github/keraton/triptr/hotel/client/HotelProviderClientV2.kt
package com.github.keraton.triptr.hotel.client

import com.github.keraton.triptr.hotel.model.Hotel

import java.util.Arrays.asList

class HotelProviderClientV2 {

    // Static constant
    companion object {
        private val HILTON = "Hilton"
        private val NOVOTEL = "Novotel"
        private val IBIS = "Ibis"
    }

    fun find(city: String): List<Hotel> {
        return asList(
                Hotel("H001", HILTON, city, 5),
                Hotel("H002", NOVOTEL, city, 3),
                Hotel("H003", IBIS, city, 2)
        )
    }
}
